.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: rust
+rust:
+- nightly-2016-04-11
+before_script:
+- |
+  pip install 'travis-cargo<0.2' --user &&
```

Comment:
> Should I add `--nocapture` so we can see the actual test output even if there was no failure?

Yeah, that sounds good for now.

> So you want to hate the build on this or should it be an allowed failure?

I think allowed failure is fine.

So my understanding is that the script as currently written doesn't benefit from travis-cargo but we could make use of it in the future. I'm fine with that.